ipa-core/src/report/hybrid_info.rs

```diff
@@ -1,22 +1,22 @@
-use crate::report::{hybrid::NonAsciiStringError, KeyIdentifier};
+use crate::report::{
+    hybrid::{InvalidHybridReportError, NonAsciiStringError},
+    KeyIdentifier,
+};
 
 const DOMAIN: &str = "private-attribution";
```

Comment:
`DOMAIN` is the `conversion_site_domain`? We shouldn't assume this is fixed (in certain types of queries it may come from multiple. We can assume `helper_origin` is fixed, like we [do here](https://github.com/private-attribution/ipa/blob/main/ipa-core/src/report/ipa.rs#L52). 

It likely will simplify things if `conversion_site_domain` is the only variable width element on the report, like in [`EncryptedOprfReport`](https://github.com/private-attribution/ipa/blob/main/ipa-core/src/report/ipa.rs#L354-L358).

`conversion_site_domain` is still a valid property to have on this struct, but it's also safe to load it directly from the static constant.

Reply:
Is `DOMAIN` just conversion_site_domain? The hybrid code has been assuming those were different things up until now. The [OPRF info struct](https://github.com/private-attribution/ipa/blob/dca5be774f9920dcc9a91e04418355802179f415/ipa-core/src/hpke/info.rs#L56) also appears to treat them as different things

Reply:
(talked about in person but including here for posterity) my bad - `DOMAIN` (poorly named in this context) is used for encryption. it will be a fixed string that is known in advance.